pyproject.toml

```diff
@@ -81,3 +83,23 @@ find = {}  # Scanning implicit namespaces is active by default
 
 [tool.setuptools.dynamic]
 version = {attr = "pyperformance.__version__"}
+
+[tool.mypy]
+python_version = "3.7"
+pretty = true
+enable_error_code = "ignore-without-code"
```

Comment:
This means that mypy will emit an error if you do a bare `# type: ignore` without a specific error code (e.g. `# type: ignore[import]` for import-related errors)